build/LAST_BOARD_TARGET

```diff
@@ -0,0 +1 @@
+nucleo
```

Comment:
delete this file. remenants of the old build system

Reply:
in fact, delete this build folder.

Reply:
shouldnt we add build to .gitginore

Reply:
No. Nothing uses build folder anymore with the new build infra

Reply:
No. Nothing uses build folder anymore with the new build infra